force_wfmanager/left_side_pane/mco_parameter_model_view.py

```diff
@@ -12,17 +12,39 @@ class MCOParameterModelView(ModelView):
     model = Instance(BaseMCOParameter, allow_none=False)
 
     #: The human readable name of the MCO parameter class
-    label = Str()
+    label = Property(Str, depends_on="model.name,model.type")
 
     #: Defines if the MCO parameter is valid or not
     valid = Bool(True)
 
+    #: An error message for issues in this modelview
+    error_message = Str
+
     #: Base view for the MCO parameter
     traits_view = View(
         Item("model.name"),
         Item("model.type"),
         kind="subpanel",
     )
 
+    #: Event to request a verification check on the workflow
+    verify_workflow_event = Event
+
+    @on_trait_change('model.name,model.type')
+    def parameter_change(self):
+        self.verify_workflow_event = True
+
+    def _get_label(self):
+        if self.model.name != '':
+            label_name = 'Name: '+self.model.name
```

Comment:
change the representation to match the kpi, like

Range: TYPE name
KPI: name (TARGET)